packages/overlay/README.md

```diff
@@ -345,19 +231,117 @@ When nesting multiple overlays, it is important to ensure that the nested overla
+
+```html
+<sp-button id="modal-trigger" aria-haspopup="dialog" aria-expanded="false">
+    Open Settings
+</sp-button>
+<sp-overlay trigger="modal-trigger@click" type="modal">
+    <sp-dialog-wrapper
+        headline="Settings"
+        dismissable
+        underlay
+        aria-labelledby="settings-heading"
+    >
+        <h2 id="settings-heading" slot="heading">Settings</h2>
+        <sp-field-label for="setting1">Email Notifications</sp-field-label>
+        <sp-switch id="setting1">Enable notifications</sp-switch>
+
+        <div slot="footer">
+            <sp-button
+                variant="secondary"
+                onclick="this.dispatchEvent(new Event('close', { bubbles: true }))"
+            >
+                Cancel
+            </sp-button>
+            <sp-button
+                variant="accent"
+                onclick="this.dispatchEvent(new Event('close', { bubbles: true }))"
+            >
+                Save
+            </sp-button>
+        </div>
+    </sp-dialog-wrapper>
+</sp-overlay>
+```
+
+#### Keyboard Navigation
+
+-   ESC key closes overlays in reverse order of opening
+-   TAB/Shift+TAB navigates through focusable elements within modal/page overlays
+-   Arrow keys navigate through menu items in menu overlays
+-   ENTER/SPACE activates buttons and controls
+
+#### Screen Reader Considerations
+
+-   Use `aria-haspopup` on trigger elements to indicate the type of overlay
+-   Provide descriptive labels using `aria-label` or `aria-labelledby`
+-   Use proper heading structure within overlays
+-   Ensure error messages are announced using `aria-live`
+
+Example of a tooltip with proper screen reader support:
+
+```html
+<sp-button id="help-trigger" aria-describedby="help-tooltip" label="Help">
+    <sp-icon-help slot="icon"></sp-icon-help>
+</sp-button>
+<sp-overlay trigger="help-trigger@hover" type="hint">
+    <sp-tooltip id="help-tooltip">
+        Click for more information about this feature
+    </sp-tooltip>
+</sp-overlay>
+```
+
+### Advanced Topics
+
+#### API
+
+```ts
+<sp-overlay
+    ?open=${boolean}
+    ?delayed=${boolean}
+    offset=${Number | [Number, Number]}
+    placement=${Placement}
+    receives-focus=${'true' | 'false' | 'auto' (default)
+    trigger=${string | ${string}@${string}}
+    .triggerElement=${HTMLElement}
+    .triggerInteraction=${'click' | 'longpress' | 'hover'}
+    type=${'auto' | 'hint' | 'manual' | 'modal' | 'page'}
+></sp-overlay>
+```
+
+##### API value interactions
+
+When a `triggerElement` is present, either through an ID reference established via the `trigger` attribute or by directly setting the `triggerElement` property, a chain on configuration begins:
+
+-   **if a `triggerInteraction` is available**, either through the `id@interaction` IDL on the `trigger` attribute or directly setting the `triggerInteraction` property, the `<sp-overlay>` will be bound to the resolved `triggerElement` via the `triggerInteraction` provided
```

Comment:
I wonder if putting this information into a table instead of a list may make it easier to mentally process...